### DISCUSSION TITLE
Adding multiple components

### DISCUSSION THREAD

**Discussion by MuhammadKalimullahkhan** [shadcn-ui/ui#3565] [Ideas]
please add a features to the CLI so that developers can add/download multiple components at once. Also same for overriding.

**Comment by pawix135** [comment]
You can pull multiple components by adding them as arguments separated by space like: `npx shadcn-ui@latest add dialog button form` .

**Reply by Maliksidk19** [reply]
or you can just write the command `npx shadcn-ui@latest add` and press enter it will show a list and you can select and install multiple components at once

**Reply by arhamkhnz** [reply]
If you want to install all components, use: `npx shadcn-ui@latest add --all`

**Comment by Isaacjon** [comment]
I did this:
npx shadcn@latest add avatar dialog popover resizable srcoll-area skeleton textarea tooltip
and I got this error:
Unexpected token < in JSON at position 0

**Reply by arhamkhnz** [reply]
Looks like there’s a typo in `scroll-area`. Try this:

`npx shadcn@latest add avatar dialog popover resizable scroll-area skeleton textarea tooltip`

**Reply by Isaacjon** [reply]
Wow, that works! Thank you

**Comment by RusiraJayatilake** [comment]
The 'shadcn-ui' package is deprecated. Please use the 'shadcn' package instead:

  npx shadcn@latest add --all

For more information, visit: https://ui.shadcn.com/docs/cli
